<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathFinderClass</class>
 <widget class="QMainWindow" name="PathFinderClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PathFinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 89, 89);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>775</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>load image</string>
    </property>
   </widget>
   <widget class="MapView" name="mapView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>1106</width>
      <height>766</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMapButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>775</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>load map</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoMessageBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>775</y>
      <width>411</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(190, 190, 190);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveMapButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>775</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>save map</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>775</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color:rgb(40,40,40);
	color: rgb(190, 190, 190);
	border: 1px solid rgb(120,120,120);
}
QLineEdit:disabled{
	background-color:rgb(90,90,90);
	color: rgb(190, 190, 190);
	border: 1px solid rgb(120,120,120);
}


</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>775</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(145, 145, 145);</string>
    </property>
    <property name="text">
     <string>compute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header location="global">MapView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PathFinder.qrc"/>
 </resources>
 <connections/>
</ui>
